func_adl_xAOD/cms/aod/event_collections.py

```diff
@@ -51,3 +70,7 @@ def get_running_code(self, container_type: event_collection_container) -> list:
 
 ctyp.add_method_type_info("reco::Track", "hitPattern", ctyp.terminal('reco::HitPattern'))
 ctyp.add_method_type_info("reco::Muon", "hitPattern", ctyp.terminal('reco::HitPattern'))
+ctyp.add_method_type_info("reco::Muon", "globalTrack", ctyp.terminal('reco::Track', is_pointer=True))
+ctyp.add_method_type_info("reco::Muon", "isPFMuon", ctyp.terminal('bool'))
+ctyp.add_method_type_info("reco::Muon", "isPFIsolationValid", ctyp.terminal('bool'))
+ctyp.add_method_type_info("reco::Track", "isNonnull", ctyp.terminal('bool'))
```

Comment:
This won't work, right? So we should not have it in here? This will have to be something seperate.